llvm/include/llvm/BinaryFormat/Dwarf.def

```diff
@@ -51,7 +51,7 @@
 #endif
 
 #ifndef HANDLE_DW_OP
-#define HANDLE_DW_OP(ID, NAME, VERSION, VENDOR)
+#define HANDLE_DW_OP(ID, NAME, OPERANDS, ARITY, VERSION, VENDOR)
```

Comment:
Is the number of operands ever useful without knowing the encoding of the operands?